core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -407,6 +429,11 @@ func (c *coordinator) unmarshalLogs(
 				return
 			}
 			newTransmissionLogs = append(newTransmissionLogs, unpacked)
+		default:
+			c.lggr.Info(fmt.Sprintf("Unexpected event sig: %s", hexutil.Encode(lg.EventSig)))
+			c.lggr.Info(fmt.Sprintf("expected one of: %s %s %s %s",
```

Comment:
Should these be warning or error logs?

Reply:
Good note, I think these would be better served as errors.